src/components/Login/Login.js

```diff
@@ -82,6 +89,13 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
 
   const formMessage = useMemo(() => getForgotErrorMessage(location), [location]);
 
+  const activationInProgress = useMemo(() => isActivactionInProgress(location), [location]);
+
+  if(activationInProgress && typeof(window.gtag) === 'function') {
```

Comment:
Is it possible testing it in any way?

Reply:
I mean, automatically.

Reply:
automatically? you said unit testing?

Reply:
right

Reply:
i really don't know, because the library use in the index.html. Maybe only we can simulate the response?

Reply:
Index.html cannot be tested, but I think that _Login.js_ could be tested, assuming that _Index.html_ works fine.

let us talk to discuss it if you want.

Reply:
i added the test